utils/writeable/benches/writeable.rs

```diff
@@ -72,7 +72,7 @@ writeable::impl_display_with_writeable!(ComplexWriteable<'_>);
 
 const SHORT_STR: &str = "short";
 const MEDIUM_STR: &str = "this is a medium-length string";
-const LONG_STR: &str = "this string is very very very very very very very very very very very very very very very very very very very very very very very very long";
+const LONG_STR: &str = "this is a very very very very very very very very very very very very very very very very very very very very very very very very long string";
```

Comment:
this change seems unnecessary and invalidates benchmark history.

Reply:
I thought a longer overlap would make more meaningful benchmarks for cmp_bytes

Reply:
> I thought a longer overlap would make more meaningful benchmarks for cmp_bytes

And it totally is. Just use a new const for it so all the preexisting benchmarks are kept stable.

Reply:
I don't really see the problem to be solved (the new string is only 2 bytes longer than the old one, and people who track the benchmarks over time can see this commit as the culprit) and prefer merging this as-is but I can make @Manishearth's suggested change if necessary.

Reply:
> people who track the benchmarks over time can see this commit as the culprit

If you're looking at the graph you don't know whether it's a benchmark setup change, or a behaviour change. All you'll see is "Make writeable_cmp_bytes a free function", which *shouldn't change any benchmarks*.

Reply:
I made the string the same length as before. (The bigger change requires a setup with a code editor which I don't have at the moment but will in about an hour.) I still prefer landing this to avoid the context switch later.

Reply:
ok I didn't hear back about whether my compromise was sufficient so I went ahead and implemented the solution that I believe addresses @robertbastian's blocking concern.